panda-whale-vekm/dinehard
# Mayson/database

Description
This pull request provides some styling changes including a change in font and styling done to the checkboxes.

Asana Task
N/A

Testing Instructions
Run npm start to spin up server and observe the change in styling


Checklist
N/A

Additional Notes, Images, etc.
N/A
